Makefile

```diff
@@ -841,14 +841,21 @@ libzephyr.a: $(filter-out %/lib.a, $(core-y))
 libapplication.a: $(app-y) $(libs-y) $(KBUILD_ZEPHYR_APP)
 	$(call cmd,ar_target)
 
+K_ENTRY=''
+ifeq ($(CONFIG_JAILHOUSE),y)
+	K_ENTRY='__jh_entry'
```

Comment:
hrm. I think we should perhaps make the starting symbol a config option

Reply:
Will do.